opentelemetry-user-events-logs/examples/basic-logs.rs

```diff
@@ -4,10 +4,17 @@ use opentelemetry_appender_tracing::layer;
 use opentelemetry_sdk::logs::SdkLoggerProvider;
 use opentelemetry_user_events_logs::{ExporterConfig, ReentrantLogProcessor, UserEventsExporter};
 use std::collections::HashMap;
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+use std::{thread, time::Duration};
 use tracing::error;
-use tracing_subscriber::prelude::*;
+use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt, EnvFilter, Layer};
 
 fn init_logger() -> SdkLoggerProvider {
+    let filter_fmt = EnvFilter::new("info").add_directive("opentelemetry=debug".parse().unwrap());
+    let fmt_layer = tracing_subscriber::fmt::layer().with_filter(filter_fmt);
+    let _guard = tracing_subscriber::registry().with(fmt_layer).set_default(); // Temporary subscriber active for this function
```

Comment:
not a blocker, but this feels better covered elsewhere than in a "basic" example. Every exporter has this issue, and it's better covered in a central place in upstream. (We have similar issue about shutdown logs, as it is order dependent also.)